### DISCUSSION TITLE
Modal backdrop classes

### DISCUSSION THREAD

**Discussion by apavliukov** [mdbootstrap/TW-Elements#1987] [Support from community] [answered]
Hey there!

Thanks for the awesome library! 

I wonder how can I customize the classes for `backdrop` element for `Modal` component? I see that there is a `backdropClasses` config option, but I don't know how to set it. I want to set one static class for the backdrop and then customize it in css file.

Thanks.

**Comment by juujisai** [comment, marked as answer]
Hi! At this moment, you can only customize the modal backdrop by setting styles to the `[data-te-backdrop-show]` element. You can do it via css

**Reply by apavliukov** [reply]
Thanks, hopefully this will be a customizable option someday.
